api/v3/Case/Getdetailscount.php

```diff
@@ -14,6 +14,10 @@ function civicrm_api3_case_getdetailscount($params) {
   $params['options'] = CRM_Utils_Array::value('options', $params, []);
   $params['options']['is_count'] = 1;
 
+  // Remove unnecesary parameters:
+  unset($params['return']);
```

Comment:
I'm not sure I like the idea of unsetting these parameters since ideally `getdetails` would work with them anyway. But I can see that they're not really relevant for a "count" query so I guess it's fine :man_shrugging: 

If you wanted to you could combine both `unset` calls into a single one,

```
unset($params['return'], $params['sequential']);
```

Since it's a varadic function that unsets all variables passed to it.